src/test/fuzz/fuzz.cpp

```diff
@@ -19,6 +19,28 @@
 
 const std::function<void(const std::string&)> G_TEST_LOG_FUN{};
 
+/**
+ * A copy of the command line arguments that start with `--`.
+ * First `LLVMFuzzerInitialize()` is called, which saves the arguments to `g_args`.
+ * Later, depending on the fuzz test, `G_TEST_COMMAND_LINE_ARGUMENTS()` may be
+ * called to fetch those arguments and fill `gArgs` from them.
```

Comment:
The fetching is done implicitly, so mention that instead of mentioning `gArgs`?

Reply:
Reworded to
```cpp
 * Later, depending on the fuzz test, `G_TEST_COMMAND_LINE_ARGUMENTS()` may be
 * called by `BasicTestingSetup` constructor to fetch those arguments and store
 * them in `BasicTestingSetup::m_node::args`.
```

Is it better?

Reply:
It *is called*, not *may be called*? Sounds better either way.

Reply:
It is called if the fuzz test creates a `BasicTestingSetup` from its `initialize_...()` function, not all of them do, right?

Reply:
If there is no `BasicTestingSetup`, you can't fetch them either ;)

But as I said, both new versions are equally fine.

Reply:
Ok :)